packages/portfolio-contract/tools/plan-solve.ts

```diff
@@ -272,103 +268,56 @@ const prettyJsonable = (obj: unknown): string => {
   return pretty;
 };
 
-const cplexLpOpForName = { min: '>=', max: '<=', equal: '=' };
-
-/**
- * Convert a javascript-lp-solver Model into CPLEX LP format.
- * https://lim.univ-reunion.fr/staff/fred/Enseignement/Optim/doc/CPLEX-LP/CPLEX-LP-file-format.html
- */
-const toCplexLpText = (model: LpModel) => {
-  const variableEntries = typedEntries(model.variables);
-  const makeSumExpr = attr =>
-    partialMap(variableEntries, ([varName, attrs]) => {
-      const attrValue = Object.hasOwn(attrs, attr) ? attrs[attr] : 0;
-      if (attrValue) return `${attrValue === 1 ? '' : attrValue} ${varName}`;
-    }).join(' + ');
-  // Variables [and constraints] can be named anything provided that the
-  // **name does not exceed 16 characters**.
-  const mappedAttrs = new Map();
-  const mode: 'MIN' | 'MAX' = model.opType.toUpperCase();
-  const text = `
-${mode}
-  ${model.optimize}: ${makeSumExpr(model.optimize)}
-SUBJECT TO
-  ${typedEntries(model.constraints)
-    .flatMap(([attr, constraint]) => {
-      let attrParts = attr.split('_');
-      if (attrParts[0] === 'through') {
-        attrParts[0] = 'thru';
-      } else if (attrParts[0] !== 'allow') {
-        // Replace everything after the first "_" with a unique counter.
-        attrParts = [
-          attrParts[0],
-          provideLazyMap(mappedAttrs, attr, () => mappedAttrs.size + 1),
-        ];
-      }
-      const prefix = attrParts.join('_');
-      const lines = [`\\# ${attr} ${prettyJsonable(constraint)}`];
-      for (const [opName, value] of Object.entries(constraint)) {
-        if (!Number.isFinite(value)) continue;
-        const label = `${prefix}_${opName.slice(0, 3)}`;
-        const op = cplexLpOpForName[opName];
-        lines.push(`${label}: ${makeSumExpr(attr)} ${op} ${value}`);
-      }
-      return lines;
-    })
-    .join('\n  ')}
-BINARY
-  ${Object.keys(model.binaries).join(' ')}
-GENERAL
-  ${Object.keys(model.ints).join(' ')}
-END`.trim();
-  return `${text}\n`;
-};
-
 // This operation is async to allow future use of async solvers if needed
 export const solveRebalance = async (
   model: LpModel,
   graph: RebalanceGraph,
 ): Promise<{ flows: SolvedEdgeFlow[]; detail?: Record<string, unknown> }> => {
-  const cplexLpText = toCplexLpText(model);
-  const matrixResult = await Promise.resolve(highsSolverP)
-    .then(solver => solver.solve(cplexLpText))
-    .catch(error => ({ Status: undefined, error }));
-  if (matrixResult.Status !== 'Optimal') {
-    // Switch to javascript-lp-solver for diagnostics.
-    const jsResult = jsLPSolver.Solve(model, 1e-9);
+  await null;
+  const jsResult = jsLPSolver.Solve(model, 1e-9);
```

Comment:
We no longer need to qualify the result as "js".
```suggestion
  const solution = jsLPSolver.Solve(model, 1e-9);
```